src/adafruit_circuitplayground/express.py

```diff
@@ -86,7 +86,7 @@ def tapped(self):
         """  Not Implemented!
         """
         telemetry_py.send_telemetry(TelemetryEvent.CPX_API_TAPPED)
-        raise NotImplementedError(CONSTANTS.NOT_IMPLEMENTED_ERROR)
+        utils.print_for_unimplemented_functions("tapped")
```

Comment:
Can you do the same for this function? (use __name)

Reply:
I cannot because it is a @property ):